### PR TITLE
Ready: Riak Control 2.0

### PR DESCRIPTION
**Pull request by lucperkins**
@angrycub Ready for review

### CODE REVIEW

File: source/languages/en/riak/ops/advanced/riak-control.md [hunk 1]
```diff
@@ -14,181 +14,151 @@ moved: {
...
-
-## Setting up Riak Control
+application](https://github.com/basho/riak_control), the necessary code
+for it ships with versions of Riak 1.1 and above and requires no
+additional installation steps.
 
-### Enabling SSL and HTTPS
+<div class="note">
+<div class="title">Note on SSL</div>
+We strongly recommend that you enable SSL for Riak Control. It is
```
Comment (cmeiklejohn) [inline source/languages/en/riak/ops/advanced/riak-control.md]: SSL isn't enabled by default; the SSL requirement for Riak Control is dependent on what the authentication method is configured as

Comment (lucperkins) [inline source/languages/en/riak/ops/advanced/riak-control.md]: Good catch. Fixed in commit 1dcb09d6739d4655b62d4defb68dc31bf9faa826.

Comment (cmeiklejohn) [inline source/languages/en/riak/ops/advanced/riak-control.md]: Might be useful to say that if the ``userlist`` method of authentication is selected, and ``ssl`` is disabled, Riak Control won't work.  This has come up a few times on the mailing list.

Comment (lucperkins) [inline source/languages/en/riak/ops/advanced/riak-control.md]: Addressed in 2103fa058fcbd67c979318bd2c87d861ec67a6ad

Comment (Pavanpatil0744) [reply source/languages/en/riak/ops/advanced/riak-control.md:26]: how to enable ssl for riak control
can someone please give me steps

**Pull request merged**